scripts/gendispatch.lua

```diff
@@ -133,13 +139,31 @@ for i,f in ipairs(shallowcopy(functions)) do
     end
     local newf = shallowcopy(f)
     newf.name = newname
+    if newname == "ui_try_resize" then
+      -- The return type was incorrectly set to Object in 0.1.5.
+      -- Keep it that way for clients that rely on this.
+      newf.return_type = "Object"
+    end
     newf.impl_name = f.name
     newf.noeval = true
+    newf.since = 0
     newf.deprecated_since = 1
     functions[#functions+1] = newf
   end
 end
 
+-- don't expose internal attributes like "impl_name" in public metadata
+exported_attributes = {'name', 'parameters', 'return_type', 'method',
+                       'since', 'deprecated_since'}
+exported_functions = {}
+for i,f in ipairs(functions) do
```

Comment:
`testlint` enforces that. Would be nice to have lua-lint running on all of our lua logic.